core/src/main/java/com/datastax/oss/driver/internal/core/cql/Conversions.java

```diff
@@ -15,11 +15,15 @@
  */
 package com.datastax.oss.driver.internal.core.cql;
 
+import static com.datastax.oss.protocol.internal.FrameCodec.headerEncodedSize;
+import static com.datastax.oss.protocol.internal.request.query.QueryOptions.queryFlagsSize;
```

Comment:
Nit: could you avoid static imports? They make the code harder to follow.
There are some exceptions for test files (see the contributing guidelines).